packages/ui/src/view/v2/GridCard/GridCardHeader/index.tsx

```diff
@@ -59,7 +61,7 @@ const GridCardHeader = ({
 
     return (
         <Title className={classnames(styles.cardHeader, className)} level={4} ref={headerContainerRef}>
-            <div className="rgl-drag-zone">
+            <div className={draggable ? 'rgl-drag-zone' : ''}>
```

Comment:
ici tu peux utiliser classnames au lieu d'un ternaire
```suggestion
            <div className={classnames({'rgl-drag-zone': draggable})>
```

Reply:
Je connaissais pas classnames, j'ai mis a jour avec le changement!